WebServers/HTTPSecureServerLite/HttpsProcessor.cs

```diff
@@ -593,12 +593,14 @@ private static async Task DefaultRoute(HttpContextBase ctx)
                                     version = 2;
                                 else if (absolutepath.Contains("/AirRace/"))
                                     version = 2;
-                                else if (absolutepath.Contains("/Uncharted2/"))
+                                else if (absolutepath.Contains("/Uncharted2"))
```

Comment:
For HTTPS, it is safe to remove every crypto entries and only keep version = 0, because so far, none OHS games uses the crypto on HTTPS. Probably a left over when I equalized the code.